### ISSUE TITLE
Suggestion - Bring more WL gang liveries to vehicles

### ISSUE THREAD

**Issue by AaronSmithCoding** [InZidiuZ/legacy-assets-issue-tracker#745]
### Name

Smiithy89

### Summary

Currently Lost have multiple bikes and a van with liveries of their gang. It would be cool to bring some more liveries to bikes and vans/cars to allow more customisation when riding round in groups.

Another example is Sol has the Western Fallen EDM bike but it would be cool to either have some liveries on the zombie chopper/bobbler etc

### Link

https://media.discordapp.net/attachments/1060561729551073300/1123649756200173728/image.png?width=1440&height=462

### Reason

I think customisation is a huge thing for groups and if its simple enough to add some pre existing gang images to liveries for some vehicles it would make a huge different at events in the city.

**Comment by gadjinee** [comment]
Would be great been able to customise and add the livery on some PDM bike, as mentioned in original post.

As well as having a livery for a Van as there’s Lost MC van, adding SOL or any other MC, specially if they have in case of SOL mechanic shop.

Great way to promote the business and as mentioned during any events I believe that would be used heavily

**Comment by DaxterPancake** [comment]
Majority of The Lost MC liveries are vanilla GTA. Not something we specifically added but just something that already exists in the base GTA files. The specific bike you have posted the photo of is NOT something custom made for The Lost MC. 

Adding liveries isn't the easiest thing in the world, especially if the texture unwrapping is hard to understand. It's a very rare occurance to see any type of custom wrap on a car/bike in general.

As the person who makes all the gang clothing and does all of the clothing in general, it is extremely hard to keep up with gangs coming and going in the city so it would just add a headache for the vehicle devs to have to try and keep up with this.

Personally, I feel like if SOL wants a specific car/van/bike to have their livery on it, they ask for it. Not only that, but there was a time where custom clothing & cars wasn't a thing and there were multiple MCs who could easily have customisation when riding around in groups. You'd easily be able to tell SOL from The Lost MC for example.

**Comment by gadjinee** [comment]
Totally understandable with the gangs, specially with them constantly coming and going.

However could this be done with stable businesses ? Like for example some tow trucks have mechanic liveries (MLT truck), but for any mechanic companies the livery could be also added to vans, like Speedo Van from EDM, it does have some of the liveries but would be cool to have like a mechanic shops there as well (Hayes, SOL, Mirror Park, Lost, Mosleys, Harmony)

**Comment by MrAngryCarver** [comment]
I second the tow trucks with mechanic shop liveries. Mechanic shops are a more permanent and less likely to change feature than gangs in the server. Having tow trucks with shop logos would help with shop advertisement and showing shop activity

**Comment by gadjinee** [comment]
Agreed, would be great to have the possibility to add mechanic shop logo to any of the tow trucks as well as some of the vans

**Comment by gadjinee** [comment]
Hi, is there any updates on adding business liveries to some of the vans ?

**Comment by RetroCharles** [comment]
I agree with stable businesses. Would give more pride and representation for the business. Having burger shot, Bean machine vehicles and tow trucks with shop liveries makes the city more alive and vibrant. Would be neat to have all tow trucks have the ability to have their shop logos, and Vans that most mechanics use for extra material storage. IT makes sense to have some liveries on vans. In the real world there are business with vans for transportation/goods/deliveries etc. 

I believe having more business liveries on more vehicles can help the business grow, and people employed in that business will be very appreciative to represent their place of work.

**Comment by gadjinee** [comment]
Any updates on having business liveries on some of the vans ? 
Sorry for been a pain, just really looking forward to be able to use it in RP and represent the company

**Comment by gadjinee** [comment]
Is it possible to maybe add some other companies liveries not just on Vans but maybe on RS3?

Hayes, mosleys, harmony

**Comment by InZidiuZ** [comment]
This issue tracker is being shut down and replaced by Discord threads. If this issue is still relevant, create a thread on [Discord](https://discord.gg/opfw).
